### DISCUSSION TITLE
Dudas Con Eclipse sobre manejo de Packages y Carpetas

### DISCUSSION THREAD

**Discussion by heinHertz** [dds-jv/foro#103] [Consultas de herramientas] [answered]
Hola para el TPA uso eclipse Eclipse IDE for Enterprise Java Developers.  Version: 2020-03 (4.15.0)
El problema parte por lo dificil que es agregar una carpeta a un package. Quiero reagrupar la una carpeta( package ) en otro lugar, pero la agrega a una Carpeta (que no es package) abajo, en una copia de mi codigo,  no entiendo porque hace una copia dentro del mismo proyecto y desconozco las diferencias entre carpeta y package. 

![image](https://user-images.githubusercontent.com/37560557/84608226-c52b7900-ae87-11ea-8d00-70bdc92d4bc1.png)

**Comment by gastonprieto** [comment, marked as answer]
Hola @heinHertz!

No lo agrega a otra carpeta, son la misma carpeta, pero en dos vistas distintas, una te lo muestra como packages y otra como folders, o sea, por abajo los packages se implementan como carpetas.

Un par de comentarios mas ahora que veo la estructura, los tests deberían estar dentro de `src/test/java` y los resources si son de test van en `src/test/resources` o en `src/main/resources` si son archivos que necesitas cuando corras la aplicación.
